drivers/sensor/apds9306/apds9306.c

```diff
@@ -128,7 +125,18 @@ static void apds9306_worker(struct k_work *p_work)
 	}
 
 	data->light = sys_get_le24(buffer);
+	LOG_DBG("Last raw measurement: %u", data->light);
+
+	/* Based on
+	 * https://github.com/gmarti/AsyncAPDS9306/blob/master/AsyncAPDS9306.cpp#L39
+	 */
+	double gain = avago_apds9306_gain[data->gain_idx];
+	double integrationtime = avago_apds9306_measurement_period[data->resolution_idx];
```

Comment:
The `integrationtime` variable name doesn't follow `snake_case` naming convention. Please rename.